Magento 2.3.3 compatibility

vincenthornikx opened the pull request:
Updated the Transport class to use the new Magento interfaces. Skip encoding issues by using the raw content of the first body part. I tested this on a clean local installation of Magento 2.3.3.

giacmir commented:
Hi @vincenthornikx,

thanks for your contribution. Is this compatible with previous versions of Magento? If not a version constraint should be specified in composer.json

vincenthornikx commented:
Hi @giacmir,
It won't be compatible with older versions as the classes I changed were introduced in Magento 2.3.3. Therefore I updated the version constraint for magento2/magento-base to >=2.3.3.
Thanks for noticing this.

vincenthornikx commented:
While the mail seems to be properly formatted in the email client, there still seem to be some unsolved issues in the actual source. I'll re-open this pull request once I solve them

vincenthornikx commented:
I re-opened this pull request after making the previous change in https://github.com/magespecialist/m2-MSP_SMTP/pull/7/commits/5e4290532fa5b40aeb5529c88ec362cbf82a3350

vincenthornikx commented:
@giacmir any update on whether this is going to go through?
